client/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@codefreak",
+  "rules": {
+    "@typescript-eslint/explicit-module-boundary-types": "off"
```

Comment:
I think this is a good rule but this would require a lot of additional effort in this repo, this is why it is only disabled locally in this repo.